weldx/asdf/schemas/weldx.bam.de/weldx/measurement/measurement_chain-1.0.0.yaml

```diff
@@ -6,19 +6,119 @@ tag: "tag:weldx.bam.de:weldx/measurement/measurement_chain-1.0.0"
 
 title: |
   Schema that describes a measurement_chain
+description: |
+  A measurement chain describes the different steps that are performed to acquire experimental data. It collects all
+  relevant information about the initial source signal and the involved transformations. In this schema, the fundamental
+  data representation of the measurement chain is a directed graph with the signals being the nodes and the
+  transformations being the edges.
+
+example:
+  -
+    - A measurement chain containing a single transformation that performs an AD conversion.
+    - |
+      !<tag:weldx.bam.de:weldx/measurement/measurement_chain-1.0.0>
+        name: Measurement Chain
+        data_source: !<tag:weldx.bam.de:weldx/measurement/source-1.0.0>
+          name: Source
+          output_signal: &id001 !<tag:weldx.bam.de:weldx/measurement/signal-1.0.0>
+            signal_type: analog
+            unit: V
+          error: !<tag:weldx.bam.de:weldx/measurement/error-1.0.0>
+            deviation: !unit/quantity-1.1.0 {unit: percent, value: 0.2}
+        source_signal: !<tag:weldx.bam.de:weldx/core/graph/di_node-1.0.0>
+          name: Source
```

Comment:
I think it would be helpful to provide a simplified real world example like `welding current` instead of a more generic/abstract one since the schema is rather complex

Reply:
I have updated my generation script. But I will wait with the update until the changes to the schemas are applied

Reply:
done